server/graphql/v2/object/Activity.ts

```diff
@@ -93,7 +93,7 @@ export const Activity = new GraphQLObjectType({
       type: new GraphQLNonNull(GraphQLJSON),
       description: 'Data attached to this activity (if any)',
       async resolve(activity, _, req: express.Request): Promise<Record<string, unknown>> {
-        const toPick = [];
```

Comment:
Since it's always going to be there, could we rather expose it as an official non-nullable attribute?

```es6
    isSystem: {
      type: new GraphQLNonNull(GraphQLBoolean),
      description: '...',
      resolve: activity => Boolean(activity.data?.isSystem)
    },
```

Reply:
Thanks makes sense. 👍🏽